src/classes.py

```diff
@@ -190,6 +192,15 @@ def take_action(self, time_elapsed, car_env, autonomous_env=None):
         self.position = self.position + self.current_speed * time_elapsed
 
 
+'''
+Truck class that extends Car.
+Truck itself do not make any decision, it has to ask the driver. (Similar to Car class)
+'''
+class Truck(Car):
+    def __init__(self, initial_speed, lane, drivers_mood=0.95, number=0, acc=0, breaking=0):
+        super().__init__(initial_speed, lane, drivers_mood, number, acc, breaking)
+        self.color = 3 # Red 
+
```

Comment:
We have to limit max speed of the truck to 90km/h or 85km/h, so i would modify the driver decide function to change the always speed_limit parameter to 90 and call the same function from parent class